os-eap-migration/added/launch/openshift-migrate-common.sh

```diff
@@ -16,11 +16,16 @@ function runMigration() {
   cp -f ${STANDALONE_XML_COPY} ${STANDALONE_XML}
 
   # exposed by wildfly-cekit-modules
+  createConfigExecutionContext
```

Comment:
@ochaloup , I am wandering if we should add these lines to configure_server_with_cli instead of adding them here. It would be there:https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch-config/config/added/launch/launch.sh#L34

Reply:
@jfdenise ok, I see. I will do so.
Do you think I may add there the configuration of the drivers there too?
https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch-config/config/added/launch/openshift-common.sh#L70

Reply:
I think that all that is needed for the migration to configure the server should be in. This method is exposed for the migration only.

Reply:
ok. I removed the code from here and I created a new issue https://issues.redhat.com/browse/CLOUD-3520 and created a PR under `wildfly-cekit-modules` here https://github.com/wildfly/wildfly-cekit-modules/pull/165

Reply:
@ochaloup , thank-you, I merged it.